Package.swift

```diff
@@ -123,7 +127,9 @@ let package = Package(
       ],
       path: "Sources/Overlays/_Testing_Foundation",
       exclude: ["CMakeLists.txt"],
-      swiftSettings: .packageSettings
+      swiftSettings: .packageSettings + [
+        .enableLibraryEvolution(applePlatformsOnly: true),
```

Comment:
Why Apple only?

Reply:
Because the Foundation module on Apple platforms has Library Evolution enabled, but on all other platforms it doesn't. Since our `_Testing_Foundation` module _publicly_ imports Foundation we get errors on non-Apple platforms due to this.

Reply:
Sigh, okay.

Reply:
Please add a comment to that effect.